keras/preprocessing/image.py

```diff
@@ -218,6 +218,8 @@ class ImageDataGenerator(object):
             It defaults to the `image_dim_ordering` value found in your
             Keras config file at `~/.keras/keras.json`.
             If you never set it, then it will be "th".
+        interpolation_order: The order of the spline interpolation, default is 0.
+            The order has to be in the range 0-5.
```

Comment:
https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.affine_transform.html
is only explaining like that.

What should I write?

> 0 is nearest-neighbor interpolation, and 1 is linear interpolation.
> Read https://en.wikipedia.org/wiki/Interpolation and https://en.wikipedia.org/wiki/Spline_interpolation for the detail.

Something like that?